lucene/core/src/java/org/apache/lucene/index/DocumentsWriter.java

```diff
@@ -474,49 +474,11 @@ long updateDocuments(final Iterable<? extends Iterable<? extends IndexableField>
     return seqNo;
   }
 
+
   long updateDocument(final Iterable<? extends IndexableField> doc, final Analyzer analyzer,
```

Comment:
Can we also remove this method and delegate `updateDocument` to `updateDocuments` in IndexWriter instead?